app/src/main/java/com/fstyle/androidtrainning/screen/detailsmovie/DetailsMovieActivity.java

```diff
@@ -32,4 +86,29 @@ protected void onStop() {
         mPresenter.onStop();
         super.onStop();
     }
+
+    @Override
+    public Integer getMovieId() {
+        return getIntent().getIntExtra(Constant.EXTRA_MOVIE_ID, 0);
+    }
+
+    @Override
+    public void onDetailsMovieSuccess(Movie movie, List<String> listGenre) {
+        String urlBackdrop = StringUtils.convertPosterPathToUrlPoster(movie.getBackdropPath());
+        String urlPoster = StringUtils.convertPosterPathToUrlPoster(movie.getPosterPath());
+        String genresCommaSeparated = StringUtils.convertListToStringCommaSeparated(listGenre);
+        Date date =
+                DateTimeUtils.convertStringToDate(movie.getReleaseDate(), DATE_FORMATT_DD_MM_YYYY);
+        String releaseDate =
+                DateTimeUtils.getStrDateTimeFormatted(date, DATE_FORMATT_DD_MMMMM_YYYY);
+
+        Glide.with(this).load(urlBackdrop).into(mImageBackdrop);
+        Glide.with(this).load(urlPoster).into(mImagePoster);
+        mTextReleaseDate.setText(releaseDate);
+        mTextGenreMovie.setText(genresCommaSeparated);
+        mTextRateMovie.setText(movie.getVoteAverage() + MAX_POINT);
```

Comment:
WARNING: [SetTextI18n] Do not concatenate text displayed with `setText`. Use resource string with placeholders.
 When calling `TextView#setText`
* Never call `Number#toString()` to format numbers; it will not handle fraction separators and locale-specific digits properly. Consider using `String#format` with proper format specifications (`%d` or `%f`) instead.
* Do not pass a string literal (e.g. "Hello") to display text. Hardcoded text can not be properly translated to other languages. Consider using Android resource strings instead.
* Do not build messages by concatenating text chunks. Such messages can not be properly translated.